mlrun/model_monitoring/controller.py

```diff
@@ -265,9 +268,11 @@ def _get_model_monitoring_access_key() -> Optional[str]:
             access_key = mlrun.mlconf.get_v3io_access_key()
         return access_key
 
-    @staticmethod
     def _should_monitor_endpoint(
-        endpoint: mlrun.common.schemas.ModelEndpoint, application_names: set
+        self,
+        endpoint: mlrun.common.schemas.ModelEndpoint,
+        application_names: set,
+        base_period_minutes: int,
```

Comment:
I know we should do it before, but please have full explanation on this functionality (when we should or not monitor)